apps/api-gateway/src/issuance/issuance.controller.ts

```diff
@@ -713,17 +733,24 @@ async downloadBulkIssuanceCSVTemplate(
       name:'credentialType',
       enum: IssueCredentialType
     })
+    @ApiQuery({
+      name: 'isValidateSchema',
+      type: Boolean,
+      required: false
+    })
     @UseGuards(AuthGuard('jwt'), OrgRolesGuard)
     @Roles(OrgRoles.OWNER, OrgRoles.ADMIN, OrgRoles.ISSUER)
     @ApiResponse({ status: HttpStatus.CREATED, description: 'Success', type: ApiResponseDto })
     async createOOBCredentialOffer(
       @Query('credentialType') credentialType: IssueCredentialType = IssueCredentialType.INDY,
+      @Query('isValidateSchema') isValidateSchema: boolean = true,
```

Comment:
```suggestion
      @Query('isValidateSchema') isValidateSchema: boolean = true,
```We can add the this parameter in body

Reply:
I kept it in the query parameter for the API's consistency.